src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -444,7 +458,7 @@ const AccessibilityRequestDetailPage = () => {
             </Breadcrumb>
             <Breadcrumb current>{requestName}</Breadcrumb>
           </BreadcrumbBar>
-          {(message || noteMutationError) && (
+          {(message || noteMutationError) && !returnedUserErrors && (
```

Comment:
This is pretty confusing. If it works, we can merge it in and address it separately

Reply:
I did a refactor